packages/account-v2/src/containers/LoginHistoryTable/LoginHistoryTable.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { LoginHistory } from '@deriv/api-types';
+import { Table } from '@deriv-com/ui';
+import { useFormattedLoginHistoryData } from '../../hooks/useFormattedLoginHistoryData';
+
+type TLoginHistoryData = {
+    loginHistory: LoginHistory;
+};
+
+const headers = {
+    action: 'Action',
+    browser: 'Browser',
+    datetime: 'Date and Time',
+    ipAddress: 'IP Address',
+    status: 'Status',
+};
+
+const columnOrder = ['datetime', 'action', 'browser', 'ipAddress', 'status'] as const;
+
+export const LoginHistoryTable = ({ loginHistory }: TLoginHistoryData) => {
+    const columns = columnOrder.map(key => ({ header: headers[key] }));
+    return (
+        // formatedUtils
+        <div className='flex flex-col'>
+            <Table
+                columns={columns}
+                data={useFormattedLoginHistoryData(loginHistory) ?? []}
```

Comment:
looks more like a helper function, however if this is a hook, please follow the hook usage pattern

```
const formattedLoginHistory = useMemo(() => useFormattedLoginHistoryData(loginHistory) ?? [], [loginHistory])


return(
  ... 
  data={formattedLoginHistory}
  ...
)